turf/bq/doc/REFERENCE.md

```diff
@@ -0,0 +1,96 @@
+## Reference
+
+### TURF
+
+For more detailed reference on turf visit https://turfjs.org/docs/.
+#### Examples:
+
+```
+ST_BUFFER(geometry_to_buffer GEOGRAPHY, radius NUMERIC, units STRING, steps NUMERIC)
+```
+As taken from https://turfjs.org/docs/#buffer
+
+``` sql
+SELECT jslibs.turf.ST_BUFFER(ST_GEOGPOINT(-74.00,40.7128),1,'kilometers',10) as geo
+```
+
+#### VERSION
+
+{{% bannerNote type="code" %}}
+turf.VERSION()
+{{%/ bannerNote %}}
+
+Returns the current version of the turf library.
+
+#### BBOX
+
+{{% bannerNote type="code" %}}
+turf.BBOX(geojson STRING)
+{{%/ bannerNote %}}
+
+Takes a set of features, calculates the bbox of all input features, and returns a bounding box (minX, minY, maxX, maxY).
+https://turfjs.org/docs/#bbox
+
+* `geojson`: `STRING` geojson to extract the bounding box from.
+
+#### BUFFER
+
+{{% bannerNote type="code" %}}
+turf.BUFFER(geojson STRING,radius NUMERIC, options STRUCT<unit STRING,steps NUMERIC>)
```

Comment:
Instead of `NUMERIC` which doesn't make sense to interface with JS, let's use `INT64` or `FLOAT64` or any numeric type more appropiate to this situation.